neuromation/api/abc.py

```diff
@@ -99,10 +99,16 @@ class ImageProgressPush:
     dst: RemoteImage
 
 
+@dataclass(frozen=True)
+class JobProgressCommit:
+    job: str
+    dst: RemoteImage
+
+
 @dataclass(frozen=True)
 class ImageProgressStep:
     message: str
-    layer_id: str
+    layer_id: str = ""
```

Comment:
Why the attr becomes optional?
What is the use case for progress without layer?

Reply:
to use it for docker-commit messages as well: 
- `{"status": f"Committing container {cont_id} as image {name}:{tag}"}`
- `{"status": "Committed"}`

Reply:
Please don't, you are mixing two independent message types.

Reply:
ok